### ISSUE TITLE
Publish test execution reports in Gradle Actions?

### ISSUE THREAD

**Issue by szpak** [gradle-nexus/publish-plugin#170]
Test execution failures might be tricky to explain:
![image](https://user-images.githubusercontent.com/148013/219484205-1bb7ed9e-90f9-4b98-844a-5a676352f883.png)

![image](https://user-images.githubusercontent.com/148013/219484496-57092da5-00a3-4b1c-b012-4b04e686d55c.png)

Displaying a stacktrace on CI is some solution, but one the other hand a test execution report could be published (for test and compactTest*, I don't know about the e2eTest which are optional) for example with https://github.com/marketplace/actions/test-reporter (I haven't been using it before).

**Comment by szpak** [comment]
I'm not sure, if we still need it, having the Gradle scan in place on CI server. @TWiStErRob WDYT?

**Comment by TWiStErRob** [comment]
I really like this view and use it on all my projects: https://github.com/TWiStErRob/net.twisterrob.gradle/runs/15212418581

accessible from a PR/commit with one click:
![image](https://github.com/gradle-nexus/publish-plugin/assets/2906988/9067ad3b-be18-4ddf-995b-2f79d8c3492d)

I would go insane without it, logs are hard to parse; artifacts take long to download, extract, open; Gradle Build scans need activation, and quite heavy for this use case. I think for contributors the best place for telling them what's wrong is right in the PR. (The action has capabilities to put the failure right on the test method or inside it, it just needs some XML massaging to achieve this which I haven't got around to yet.)
